css/amp.css

```diff
@@ -328,6 +328,20 @@ template {
   box-sizing: border-box;
 }
 
+/**
+ * Authors can add this class to their html tags to provide `border-box` box-sizing
+ * can be easily overriden when you are using a third party component.
+ * https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/
+ */
+.amp-border-box-inherit {
```

Comment:
I don't think it's necessary to add this as well. Just having a `.amp-box-sizing-inherit` will allow give you the same benefits and you can define your own custom root `bos-sizing` to be inherited.